src/performance_profiles.jl

```diff
@@ -61,7 +61,8 @@ function performance_profile(T :: Array{Float64,2}, labels :: Vector{AbstractStr
   xs = [1:np+1;] / np
   length(labels) == 0 && (labels = [@sprintf("column %d", col) for col = 1 : ns])
   xlabel = haskey(kwargs, :xlabel) ? pop!(kwargs, :xlabel) : "Within this factor of the best" * (logscale ? " (log scale)" : "")
-  ylabel = haskey(kwargs, :ylabel) ? pop!(kwargs, :ylabel) : ylab = "Proportion of problems"
+  ylabel = haskey(kwargs, :ylabel) ? pop!(kwargs, :ylabel) : "Proportion of problems"
+  linestyles = haskey(kwargs, :linestyles) ? pop!(kwargs, :linestyles) : Symbol[]
```

Comment:
I just found out we have `pop!(collection, key, default)`, which should simplify these a lot.

Reply:
Indeed, and let me also add a test.

Reply:
I just found out you can pass `linestyle=:auto` and Plots.jl cycles through the linestyles. Oh well :-|